resources/cmake/Modules/ConnextDdsGenerateSecurityArtifacts.cmake

```diff
@@ -38,87 +38,183 @@ Note regarding wolfSSL:
+            Permissions2
+    )
+
+    set(artifacts_input_files)
+    list(APPEND artifacts_input_files
         "${ca_config_file}"
         "${peer1_config_file}"
         "${peer2_config_file}"
     )
-    set(xmls_name Governance Permissions1 Permissions2)
     foreach(xml ${xmls_name})
-        list(APPEND artifacts_input_files "${xml}.xml")
+        list(APPEND artifacts_input_files
+            "${ARGS_OUTPUT_DIRECTORY}/xml/${xml}.xml"
+        )
     endforeach()
+    set(dependencies ${dependencies} ${artifacts_input_files} PARENT_SCOPE)
 
+    # ##########################################################################
+    # Copy the source files from the resource folder common for all examples
+    # (ARGS_RESOURCE_DIRECTORY) to the binary folder for the particular example.
+    # ##########################################################################
     add_custom_command(
         OUTPUT ${artifacts_input_files}
         PRE_BUILD
         COMMENT "Copying security resources to the example's binary directory"
-        COMMAND
-            ${CMAKE_COMMAND} -E make_directory ${artifacts_output_dir}
-        COMMAND # ecdsa01. Copy config files from the repository resources.
+        COMMAND # Copy config files from the repository resources.
             ${CMAKE_COMMAND} -E copy_directory
-                    "${resources_input_dir}/ecdsa01"
-                    "${artifacts_output_dir}/ecdsa01"
+                    "${ARGS_RESOURCE_DIRECTORY}/${ARGS_CA_DIRECTORY}"
+                    "${ARGS_OUTPUT_DIRECTORY}"
         COMMAND # xml. Copy Governance/Permissions from the repository resources.
             ${CMAKE_COMMAND} -E copy_directory
-                "${resources_input_dir}/xml"
-                "${openssl_working_dir}/xml"
+                "${ARGS_RESOURCE_DIRECTORY}/xml"
+                "${ARGS_OUTPUT_DIRECTORY}/xml"
         COMMAND
             ${CMAKE_COMMAND} -E make_directory ${signed_xml_output_dir}
         DEPENDS
-            ${resources_input_dir}
+            ${ARGS_RESOURCE_DIRECTORY}
     )
 
     # ##########################################################################
     # Openssl configuration.
     # ##########################################################################
     # Set output files.
-    set(ca_key_file "${certificates_output_dir}/ca_key.pem")
-    set(ca_cert_file "${certificates_output_dir}/ca_cert.pem")
-    set(peer1_key_file "${certificates_output_dir}/peer1_key.pem")
-    set(peer1_cert_file "${certificates_output_dir}/peer1_cert.pem")
-    set(peer2_key_file "${certificates_output_dir}/peer2_key.pem")
-    set(peer2_cert_file "${certificates_output_dir}/peer2_cert.pem")
-    set(artifacts_output_files
+    set(ca_key_file "${certificates_output_dir}/caKey.pem")
+    set(ca_cert_file "${certificates_output_dir}/caCert.pem")
+    set(peer1_key_file "${certificates_output_dir}/peer01Key.pem")
+    set(peer1_cert_file "${certificates_output_dir}/peer01Cert.pem")
```

Comment:
Renaming the certificates will require updating one line in the `UseOpenSSL.cmake` file of the `rticommunity/rticonnextdds-cmake-utils` repository.
See line [in this comment](https://github.com/rticommunity/rticonnextdds-cmake-utils/pull/68#discussion_r1006700708).

Reply:
Not true any more. `UseOpenSSL.cmake` won't require the certificate name to have a certain suffix